.github/workflows/integrate.yaml

```
@@ -77,25 +77,13 @@ jobs:
         if: failure()
 
       - name: Get oidc-gatekeeper workload logs
-        run: kubectl logs --tail 100 -nci-test -ljuju-app=oidc-gatekeeper
-        if: failure()
-
-      - name: Get oidc-gatekeeper operator logs
-        run: kubectl logs --tail 100 -nci-test -ljuju-operator=oidc-gatekeeper
+        run: kubectl logs --tail 100 -ntesting -l"operator.juju.is/name=oidc-gatekeeper"
```

Since we're touching this, there's no reason for shortcuts, really:
```suggestion
        run: kubectl logs --tail 100 -n testing -l "operator.juju.is/name=oidc-gatekeeper"
```